app/src/main/res/layout-sw600dp-port/feedback_item.xml

```diff
@@ -12,6 +12,22 @@
       type="org.oppia.android.app.player.state.itemviewmodel.FeedbackViewModel" />
   </data>
 
+  <!-- EXPLORATION SPLIT VIEW -->
```

Comment:
@Mehul-Bisht 
nit changes
maintain either All Capitals or only First Word Capital
Half of the files here have All Capital And the remaining half have Only first world capital

Reply:
The files which are in `layout` folder contains caps to make it focusable easily and `dimen` files contains first captial as its just normal comments.